app/javascript/components/admin/roles/forms/EditRoleForm.jsx

```diff
@@ -41,13 +43,64 @@ export default function EditRoleForm({ role }) {
         </Form>
         {(roomConfigs.isLoading || rolePermissions.isLoading || roomConfigs.data.record === 'optional') && (
           <Stack>
+            <RolePermissionRow
+              permissionName="CreateRoom"
+              description="Can create rooms"
+              roleId={role.id}
+              defaultValue={rolePermissions.data.CreateRoom === 'true'}
```

Comment:
Maybe we could remove the .data from the admin/role_permissions controller in this PR?

Reply:
The `.data` is the actual data that is returned from the `GET` request done in `useRolePermissions` (in this case it's the hash).
I see this is the way we are doing it in other places, the alternative would be to do this:
```
const { data: rolePermissions, isLoading } = useRolePermissions(role.id);
rolePermissions.CreateRoom === 'true' ...

```

This is one of those things where we need to discuss which format we should stick to

Reply:
If we replace ` const rolePermissions = useRolePermissions(role.id);` with  ` const {data: rolePermissions, isLoading: rolePermissionsIsLoading = useRolePermissions(role.id);` wouldn't that solve our problem?

Reply:
Yeah it should work, just we have 2 different ways of doing things like this, that's why I was saying we should agree on one approach. I guess we can do the way mentioned now on and later on change the places where the .data is used

Reply:
Right right we are renaming data when destructuring the object

Reply:
Agreed